### DISCUSSION TITLE
Support for HEIC/HEIF images

### DISCUSSION THREAD

**Discussion by Ranbato** [jonaswinkler/paperless-ng#1355] [Feature Requests]
I recently tried to process a page that I took a photo of from my iPhone.  Of course, the default format is HEIF and isn't recognized.
It seems like it would be moderately straightforward to add an imagemagick step to convert any heif, heic, webp, etc. files into one of the supported formats and then process them.
In the current Docker images it looks like the version installed supports converting those formats
```
# convert --version
Version: ImageMagick 6.9.11-60 Q16 x86_64 2021-01-25 https://imagemagick.org
Copyright: (C) 1999-2021 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC Modules OpenMP(4.5)
Delegates (built-in): bzlib djvu fftw fontconfig freetype heic jbig jng jp2 jpeg lcms lqr ltdl lzma openexr pangocairo png tiff webp wmf x xml zlib
#
```

**Comment by lenaxia** [comment]
I think this is a pretty big item because of how common of a use case it is to use your phone to take a picture of something

**Comment by mpfl** [comment]
A temporary workaround is to get a phone app like Genius Scan to process the existing photo.
